backend/src/db/migrations/20230826083159_add_user_roles_table.ts

```diff
@@ -0,0 +1,44 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'user_roles';
+
+const ColumnName = {
+  ID: 'id',
+  NAME: 'name',
+  KEY: 'key',
+  CREATED_AT: 'created_at',
+  UPDATED_AT: 'updated_at',
+} as const;
+
+const ROLES = [
+  {
+    id: 1,
+    name: 'Chatbot',
+    key: 'chatbot',
+  },
+  { id: 2, name: 'User', key: 'user' },
+];
+
+async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable(TABLE_NAME, (table) => {
+    table.increments(ColumnName.ID).primary();
+    table.string(ColumnName.NAME).notNullable();
+    table.string(ColumnName.KEY).checkIn(['chatbot', 'user']).notNullable();
```

Comment:
Create an enum with user roles and use `table.enum(ColumnName.KEY, Object.values(YOUR_ENUM))`
Do the same for Name column